style.css

```diff
@@ -2332,51 +2333,50 @@ object {
 ## Galleries
 --------------------------------------------------------------*/
 
-.gallery {
-	margin-bottom: 1.5em;
-}
-
 .gallery-item {
 	display: inline-block;
-	text-align: center;
+	text-align: left;
 	vertical-align: top;
-	width: 100%;
-}
-
-.gallery-columns-2 .gallery-item {
-	max-width: 50%;
-}
-
-.gallery-columns-3 .gallery-item {
-	max-width: 33.33%;
+	margin: 0 0 1.5em;
+	padding: 0 1em 0 0;
+	width: 50%;
 }
 
-.gallery-columns-4 .gallery-item {
-	max-width: 25%;
+.gallery-columns-1 .gallery-item {
+	width: 100%;
 }
 
-.gallery-columns-5 .gallery-item {
-	max-width: 20%;
+.gallery-item a {
+	display: inline-block;
+	border-bottom: none;
+	background: none;
 }
 
-.gallery-columns-6 .gallery-item {
-	max-width: 16.66%;
+.gallery-item a:hover {
+	background: none;
 }
 
-.gallery-columns-7 .gallery-item {
-	max-width: 14.28%;
+.gallery-item a img {
+	display: block;
+	transition: filter 0.2s ease-in;
+	-webkit-backface-visibility: hidden;
+	backface-visibility: hidden;
 }
 
-.gallery-columns-8 .gallery-item {
-	max-width: 12.5%;
+.gallery-item a:hover img {
+	filter: opacity(60%);
+	
 }
 
-.gallery-columns-9 .gallery-item {
-	max-width: 11.11%;
+.gallery-columns-2 .gallery-item {
```

Comment:
I think this ruleset should live up near `.gallery-columns-1 .gallery-item` (or that one should be down here, either way they should be together)